### PR TITLE
Mój nowy feature

### PR DESCRIPTION
**Pull request by pawelbiesiada**

### CODE REVIEW

File: CSharpConsole/TestClass.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpConsole
+{
+    internal class TestClass
+    {
+        public void Test()
```
Comment (kryman27) [inline CSharpConsole/TestClass.cs:11]: This method should be internal.

File: CSharpConsole/Program.cs [hunk 2]
```diff
@@ -15,7 +15,8 @@ static void Main(string[] args)
 
         private static void CurrentDomain_UnhandledException(object sender, UnhandledExceptionEventArgs e)
         {
-
```
Comment (kryman27) [inline CSharpConsole/Program.cs:18]: Leave a linebreak.

**Comment by kryman27** [comment]
Where are unit tests?

**Review by kryman27 — CHANGES_REQUESTED** [review]

File: CSharpConsole/Program.cs (hunk 2 above)
Comment (pawelbiesiada) [reply CSharpConsole/Program.cs:18]: Done

File: CSharpConsole/TestClass.cs (hunk 1 above)
Comment (pawelbiesiada) [reply CSharpConsole/TestClass.cs:11]: OK, done

**Comment by pawelbiesiada** [comment]
Unit tests are no needed here

**Review by kryman27 — APPROVED** [review]

**Pull request merged**